doc-content/enterprise-only/springboot/bus-app-quartz_proc.adoc

```diff
@@ -0,0 +1,100 @@
+[id='bus-app-quartz_{context}']
+= Configuring the business application for a cluster using Quartz
+
+If you plan to run your application in a cluster you must configure the Quartz timer service.
+
+.Prerequisites
+* You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[business application] web site, that you want to use in a cluster.
+
+.Procedure
+
+. Include the following mandatory parameters on the classpath
+or on the file system (if the path is given):
++
+[source, bash]
+----
+jbpm.quartz.enabled=true
+jbpm.quartz.configuration=quartz.properties
+----
+
+. Create the `quartz.properties` file, add the following content, and add the file path to the classpath
```

Comment:
@MarianMacik I updated as suggested, and moved the create quartz.properties to be first.